vagrant/provisioning/arkcase-ce-facts.yml

```diff
@@ -480,3 +480,7 @@ email_receiver_complaint_password: ""
 email_receiver_consultation_account: ""
 email_receiver_consultation_account_escaped: "{{ email_receiver_consultation_account | replace('@', '%40') }}"
 email_receiver_consultation_password: ""
+
+# define username and password for ActiveMQ authentication between apps
+activemq_user: ""
```

Comment:
Consider one user id for core ArkCase; another user id for config-server; and another user id for Alfresco.  In this way, eventually we could configure privileges on each queue specific to each client.